dao/src/test/java/com/carhouse/dao/impl/CarSaleDaoImplTest.java

```diff
@@ -88,13 +91,24 @@ void updateCarSaleWithWrongReference() {
     @Test
     void deleteCarSale() {
         int size = carSaleDao.getCarSales().size();
-        carSaleDao.deleteCarSale(3);
+        boolean isDeleted = carSaleDao.deleteCarSale(3);
+        assertTrue(isDeleted);
         assertEquals(size - 1, carSaleDao.getCarSales().size());
         assertThrows(EmptyResultDataAccessException.class, () -> carSaleDao.getCarSale(3));
```

Comment:
what for? You has already checked that size less than before deleting.

Reply:
I check or deleteCarSale method returns the correct result

Reply:
And what is correct result here and why exactly it?

Reply:
The correct result is true
Intellij Idea says what it is better to use than assertEquals
I removed the size check because I think it’s enough to check if this car exists